### ISSUE TITLE
Vue Component cannot used in package

### ISSUE THREAD

**Issue by nizulzaim** [Akryum/meteor-vue-component#35]
Hi I tried to create package from vue component. However, i dont think it is possible to create package using vue-component. Here is the error in console.

```
 While processing files with akryum:vue-component (for target web.browser):
   fs.js:1031:11: watch ENOENT
   at errnoException (fs.js:1031:11)
   at FSWatcher.start (fs.js:1063:11)
   at Object.fs.watch (fs.js:1088:11)
   at packages/vue-component/plugin/vue-compiler.js:228:24
   at VueComponentCompiler.addCompileResult (packages/vue-component/plugin/vue-compiler.js:151:15)
   at packages/caching-compiler/caching-compiler.js:305:14
   at C:\Users\Nizul Zaim\AppData\Local\.meteor\packages\akryum_vue-component\0.3.7\plugin.vue-component.os\npm\node_modules\meteor\caching-compiler\node_modules\async\lib\async.js:182:20
   at replenish (C:\Users\Nizul Zaim\AppData\Local\.meteor\packages\akryum_vue-component\0.3.7\plugin.vue-component.os\npm\node_modules\meteor\caching-compiler\node_modules\async\lib\async.js:317:21)
   at C:\Users\Nizul Zaim\AppData\Local\.meteor\packages\akryum_vue-component\0.3.7\plugin.vue-component.os\npm\node_modules\meteor\caching-compiler\node_modules\async\lib\async.js:328:15
   at Object.async.forEachLimit.async.eachLimit (C:\Users\Nizul Zaim\AppData\Local\.meteor\packages\akryum_vue-component\0.3.7\plugin.vue-component.os\npm\node_modules\meteor\caching-compiler\node_modules\async\lib\async.js:221:35)
   at VueComponentCompiler.processFilesForTarget (packages/caching-compiler/caching-compiler.js:277:11)
```

And here is packages.js 

```
Package.onUse(function (api) {
    api.versionsFrom('1.3.3');
    api.use('ecmascript');
    api.use(['akryum:vue@1.0.2', 'akryum:vue-component@0.3.1']);
    api.mainModule('xxx-core.js');
    api.addFiles([
        'lib/css/xxx.css',
    ], 'client');
  });
```

If you notice it, there are no .vue or .global.vue file.

**Comment by Akryum** [comment]
What is the content of your `xxx-core.js` file?
Did you try `api.mainModule('xxx-core.js', 'client');`?

**Comment by nizulzaim** [comment]
This is my content in xxx-core.js

```
import { Vue } from 'meteor/akryum:vue';
export const name = 'xxx-core';
```

I have updated my packages.js to below code but nothing change, the error still there : 

```
Package.onUse(function (api) {
    api.versionsFrom('1.3.3');
    api.use('ecmascript');
    api.use(['akryum:vue@1.0.2', 'akryum:vue-component@0.3.7'], 'client');
    api.mainModule('xxx-core.js', 'client');
    api.addFiles([
        'lib/css/xxx.css',
    ], 'client');
});
```

**Comment by nizulzaim** [comment]
Di u try to make one new packages that use this two packages? If yes, can I see your code?

**Comment by Akryum** [comment]
```javascript
Package.describe({
  name: 'use-vue-component',
  version: '0.0.1',
  summary: '',
  git: '',
  documentation: 'README.md'
});

Package.onUse(function(api) {
  api.versionsFrom('1.3.3.1');
  api.use('ecmascript');
  api.use(['akryum:vue@1.0.2', 'akryum:vue-component@0.3.7']);
  api.mainModule('client/main.js', 'client');
});
```

**Comment by nizulzaim** [comment]
The error still there. Same error, maybe it is not your code problem but FS watcher problem? Becaude i run it in Windows OS

**Comment by Akryum** [comment]
Okay I was able to reproduce the issue, currently working on a fix.

**Comment by nizulzaim** [comment]
Nice! i will test it when it done.

**Comment by Akryum** [comment]
I just published akryum:vue-component@0.3.8, does it fix it?

**Comment by nizulzaim** [comment]
Yeah.. fix it.. Thank you!
